starter-code/src/app/components/log-form/log-form.component.html

```diff
@@ -0,0 +1,30 @@
+<form (ngSubmit)="submitForm(form)" #form="ngForm" novalidate>
+
+  <div class="field" [ngClass]="{'has-error': feedbackEnabled && personField.errors}">
+    <label>Person</label>
+    <input type="text" name="person" [(ngModel)]="person" #personField="ngModel" required minlength="3"/>
+    <div *ngIf="feedbackEnabled && personField.errors">
+      <p *ngIf="personField.errors.required" class="validation">person required</p>
+      <p *ngIf="personField.errors.minlength" class="validation">person is too short</p>
+    </div>
+  </div>
+
+  <div class="field" [ngClass]="{'has-error': feedbackEnabled && messageField.errors}">
+    <label>Message</label>
+    <input type="text" name="message" [(ngModel)]="message" #messageField="ngModel" required minlength="3"/>
+    <div *ngIf="feedbackEnabled && messageField.errors">
+      <p *ngIf="messageField.errors.required" class="validation">message required</p>
+      <p *ngIf="messageField.errors.minlength" class="validation">message is too short</p>
+    </div>
+  </div>
+
+  <div class="field submit">
+    <button type="submit" (click)="accessControlLogService.addAccessItem(person, message)">Login</button>
```

Comment:
Never call the service from the html template of the component. The information should be submitted with the form.